### ISSUE TITLE
FCMClient is not sending collapse key.

### ISSUE THREAD

**Issue by peterpro** [redjanym/FCMBundle#16]
Hello!

Thanks for the great bundle! 
By the way i found another problem, as far as i can see - FCMClient not sending collapse key.
FCMBundle/FCMClient.php, FCMClient::sendNotification:

```
...
        $message = (new Message())
            ->setNotification($notification)
            ->setData($notification->getData())
            ->setPriority($notification->getPriority())
            ->setJsonData($notification->getJsonData())
            ->setContentAvailable($notification->getContentAvailable());
...
```

so there is no
```
->setCollapseKey($notification->getCollapseKey);
```
statement :(

**Comment by redjanym** [comment]
Hi @peterpro,
thank you for making me know about this problem. I will fix it asap but you are also welcomed to provide a PR. 
Thanks

**Comment by peterpro** [comment]
@redjanym OK, if I'll find something else I'll do PR instead of issue.
Thanks for the immediate response!

**Comment by redjanym** [comment]
Problem fixed in tag 1.1.6 :)
